release-notes.md

```diff
@@ -1,5 +1,40 @@
 # Two-Node Toolbox Release Notes
 
+## Version 0.5.2 - KCLI Deployment Integration
+*Release Date: August 2025*
+
+### New Features
+
+#### KCLI Alternative Deployment Method
+- Added `kcli-install` role providing alternative deployment method using kcli virtualization management tool
+- Support for both fencing and arbiter topologies with kcli-based deployment
+- Automated libvirt virtualization stack and kcli installation from COPR repository
+- Integration with kcli's BMC/Redfish simulation for fencing configuration
```

Comment:
You may have answered this above, but does kcli offer specialized setup for Redfish/Sushy or did you manually add stuff to support that?